src/alpaka/AlpakaCore/alpaka/alpakaDevAcc.cc

```diff
@@ -0,0 +1,5 @@
+#include "AlpakaCore/alpakaDevAcc.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  const DevAcc1 device = alpaka::getDevByIdx<PltfAcc1>(0u);
```

Comment:
I wonder why this is needed as a global object (I may find the reason while reading code)? Anyway, ok for now as it can be improved later.

Reply:
It certainly makes some things simpler to avoid passing the device objects around. These may become easier with the `ScopedContext` etc where we can more easily hide these details without resorting to globals.

I would expect this global to require that a device for each accelerator actually exists, and that the program would not be able support multiple devices of the same type.

Reply:
>These may become easier with the ScopedContext etc where we can more easily hide these details without resorting to globals.

Yes exactly.

Reply:
This was actually an issue for me for some time, as in the examples, the host device were just passed around through 2 or 3 files.
In the implementation inside Kokkos, from what I understood, there is simply the use of a singleton pattern to access host/device. There was no such feature in Alpaka.
I eventually found it a nice workaround for now, without modifying the Acc classes inside Alpaka. But indeed, some more framework-related work will be needed to directly access these with a Context or equivalent.

Reply:
> In the implementation inside Kokkos, from what I understood, there is simply the use of a singleton pattern to access host/device

Right, because Kokkos itself provides those constraints (at most one device backend can be enabled). For the goal of supporting multiple device backends in a single build that constraint causes quite some headache.